README.md

```diff
@@ -2,78 +2,71 @@
 
 ![Build Status](https://travis-ci.org/Pitt-CSC/PittAPI.svg?branch=master)
 ![License](https://img.shields.io/badge/license-GPLv2-blue.svg)
-![Python Version](https://img.shields.io/badge/python-%3E%3D%203.6-green.svg)
+![Python Version](https://img.shields.io/badge/python-3.12-green.svg)
```

Comment:
Does this really require Python >= 3.12? It seems limiting to support only the latest version of Python with the library. Please lower this accordingly to the actual lowest supported version of Python.

Reply:
I assumed that the repo now only supports 3.12, since that's what the rest explicitly calls for 3.12 (e.g., the README, the Pipfile, and the GitHub Actions workflows). #144 is the PR in which the project was officially migrated to 3.12, and I haven't tested the code with any older versions. @nij-patel, do you know if PittAPI now _only_ supports 3.12, or is it still compatible with some older versions?

Reply:
Instead of keeping this PR blocked, how about I open a new issue for updating the supported version and leave this as work for a future PR?

Reply:
Determined actual lowest supported Python version (3.9) in #178

Reply:
Updated the Python version badge to read ">= 3.9"